index.html

```diff
@@ -120,5 +120,40 @@ <h5>Next level iceland blue bottle fingerstache:</h5>
         </ul>
       </section>
     </section>
+    <section class="contact">
+      <h2>contact</h2>
+      <form>
+        <label for="name">name</label>
+        <input type="text" id="name">
+        <label for="email">e-mail</label>
+        <input type="email" id="email">
+        <label for="topic">topic</label>
+        <input type="text" id="topic">
+      </form>
+      <img src="assets/office.jpg" alt="office building image">
```

Comment:
The `alt` attributes are tricky. I think here it could say something more helpful, for example "Hello agency headquaters". In general the readers say sth like "Picture: alt tag" so there is no need to write that this is an image/picture. 😄